test/integration/action_controller/serialization_test_case_test.rb

```diff
@@ -62,10 +48,6 @@ def test_raises_argument_error_when_asserting_with_invalid_object
         assert_match 'assert_serializer only accepts a String, Symbol, Regexp, ActiveModel::Serializer, or nil', e.message
       end
 
-      def test_does_not_overwrite_notification_subscriptions
-        get :render_template
-        assert_template "template"
```

Comment:
Asserting on templates is no longer in Rails but an add-on gem since Rails 5. We can remove this spec as it is not a feature we use and a feature that requires other functionality outside the case of this gem.